docs/diagrams/src/DataStoring/DataStoring.puml

```diff
@@ -1,52 +1,52 @@
 @startuml DateStoring
 skinparam classAttributeIconSize 0
-package student {
-
-    class StudentList {
-        {static} +masterStudentList:Student[]
-        {static} +recentlyDeletedList:Student[]
-        +studentList:Student[]
-        --
-        Methods omitted
-    }
-
-    class Student {
-        +name:String
-        -totalClassesAttended:int
-        --
-        Methods omitted
-    }
-
-    class SubjectGrade {
-        -subject:String
-        -grade:double
-        -classesAttended:int
-        --
-        Methods omitted
-    }
-
-    class Details {
-        #gender:String 
-        #phoneNumber:int
-        #lastPaymentDate:LocalDate
-        #remarks:String
-        --
-        Methods omitted
-    }
-
-    class StudentAttributes {
-        --
-        Methods omitted
-    }
-
-    StudentList o-- Student
-
-    Student "attributes"---> "1" StudentAttributes
-
-    StudentAttributes --down|> Details
-    StudentAttributes -->"0..*" SubjectGrade
-    StudentAttributes "student" --> "1" Student
+hide circle
+class StudentList {
+    {static} +masterStudentList:Student[]
+    {static} +recentlyDeletedList:Student[]
+    {static} +archiveList:Student[]
+    +studentList:Student[]
+    --
+    Methods omitted
+}
+
+class Student {
+    +name:String
+    -totalClassesAttended:int
+    --
+    Methods omitted
+}
+
+class SubjectGrade {
+    -subject:String
+    -grade:double
+    -classesAttended:int
+    --
+    Methods omitted
+}
+
+class Details {
+    #gender:String 
+    #phoneNumber:int
+    #lastPaymentDate:LocalDate
+    #remarks:String
+    --
+    Methods omitted
+}
 
+class StudentAttributes {
+    --
+    Methods omitted
 }
 
+StudentList o-- Student
+
+Student "attributes"---> "1" StudentAttributes
+
+StudentAttributes --down|> Details
+StudentAttributes -->"0..*" SubjectGrade
+StudentAttributes "student" --> "1" Student
+
+
+
```

Comment:
Looks great! Formatting seems to fit with UML directions in CS2113.

![hiroi-monster-v0-mvf4vrsbayia1.jpg](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/110646458/ad2bfc28-2e3d-4718-8939-66df4a634837)